example/android/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath 'com.android.tools.build:gradle:4.0.1'
```

Comment:
This update is the reason why CircleCI is failing

Reply:
interesting. Android studio nagged me about it so I just followed along, I'll revert